apps/backend/src/modules/activity-logs/libs/types/activity-log-creation-data.type.ts

```diff
@@ -0,0 +1,12 @@
+type ActivityLogCreationData = {
```

Comment:
don't forget about file name
```suggestion
type ActivityLogCreateItemResponseDto = {
```